components/admin/lessons/AdminLessonCard.tsx

```diff
@@ -31,7 +31,7 @@ export const AdminLessonCard: React.FC<Props> = ({
         {pendingFlaggedQuestions} Pending Questions
       </div>
 
-      <Link href={docUrl || '_blank'}>
+      <Link href={`/admin/lessons/${slug}/introduction`}>
```

Comment:
We should have a constant made for `/admin` path already. If not, please make one since its used elsewhere in the codebase as well

Reply:
Also, is the slug variable guaranteed to always exist?

Reply:
> Also, is the slug variable guaranteed to always exist?

Yeah. Every Lesson has a corresponding slug(js0-js9)